### ISSUE TITLE
Give user feedback of last mint timestamp & unmintable runes

### ISSUE THREAD

**Issue by emilcondrea** [ordinals/ord#3686]
Lots of users complaining they did not receive their runes after minting. ```/output/txid:vout``` API only shows if it contains runes at the moment, but not if it contained in past. So if the user checks that endpoint later, they have no idea if they received the mints through minting, or spent them.

We need a way to let user know if their mint was successful or the cap was already reached when the mint transaction was processed. 

There was a PR for this which I think was mistakenly closed? https://github.com/ordinals/ord/pull/3476

@casey

**Comment by cryptoni9n** [comment]
Would it satisfy your requirement if we updated the `ord wallet outputs` command to display these additional fields for each utxo in the rpcwallet?

- output txid (timestamp can be derived from this)
- amount
- confirmations
- runes
- rare sats (if used with --index-sats)

**Comment by emilcondrea** [comment]
Not exactly. I think this needs to be exposed in the API so users that minted with any wallet get this fredback.

**Comment by cryptoni9n** [comment]
> Not exactly. I think this needs to be exposed in the API so users that minted with any wallet get this fredback.

perhaps both can be done to allow for the information to be available at the local client level as well as the api?

**Comment by emilcondrea** [comment]
Yes. Wallet can query the API to show this
